src/locales/en-US/translation.json

```diff
@@ -52,6 +52,9 @@
   "states": {
     "success": "Success!"
   },
+  "errors": {
+    "patientNameRequired": "No patient name entered!"
```

Comment:
To aid in finding translations easier, I think a better spot for this error message would be in the `patients` object. 

So the file would end up something like
```json
{
  ...
  "patient": {
	...,
	"errors": {
		"patientGivenNameRequired": "Patient Given Name is required."
	}
  }
}
```